.gitignore

```diff
@@ -21,3 +21,5 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+**/*.backup.js
```

Comment:
what does this file do?

Reply:
i create some js files by new idea file so i do not commit old files